rules/autoconsent/svt.se.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "svt.se",
+  "vendorUrl": "https://www.svt.se/",
+  "runContext": {
+    "urlPattern": "^https://www\\.svt\\.se/"
+  },
+  "prehideSelectors": [".CookieConsent__root___3GjU8"],
+  "detectCmp": [
+    {
+      "exists": ".CookieConsent__root___3GjU8"
+    }
+  ],
+  "detectPopup": [
+    {
+      "visible": ".CookieConsent__modal___gvkZW"
+    }
+  ],
+  "optIn": [
+    {
+      "waitForThenClick": ".CookieConsent__primaryButton___Th47k"
+    }
+  ],
+  "optOut": [
+    {
+      "waitForThenClick": ".CookieConsent__secondaryButton___jzCXF:nth-child(2)"
```

Comment:
Again this fails, the CSS classes have changed:
![Screenshot 2024-02-01 at 12 13 40](https://github.com/duckduckgo/autoconsent/assets/248111/0d9dddf5-48f6-4edf-b448-ff6e7f9b6c7e)